Command/UpdateContentMetadataCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * File containing the CreateContentCommand class.
+ *
+ * @copyright Copyright (C) 2012 eZ Systems AS. All rights reserved.
+ * @license http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
+ * @version //autogentag//
+ */
+namespace EzSystems\CookbookBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand,
+    Symfony\Component\Console\Input\InputInterface,
+    Symfony\Component\Console\Output\OutputInterface,
+    Symfony\Component\Console\Input\InputArgument,
+    Symfony\Component\Console\Input\InputOption;
+
+class UpdateContentMetadataCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this->setName( 'ezpublish:cookbook:update_content_metadata' )->setDefinition(
+            array(
+                new InputArgument( 'contentId' , InputArgument::REQUIRED, 'the content to be updated' ),
+                new InputArgument( 'newOwnerId' , InputArgument::REQUIRED, 'the new title of the content' ),
+                new InputArgument( 'newModificationDate' , InputArgument::REQUIRED, 'the new body of the content' ),
```

Comment:
`the new body of the content` => `new modification date for the content`

Also should be made optional